### DISCUSSION TITLE
Amazing project

### DISCUSSION THREAD

**Discussion by fontexD** [johanmeijer/grott#292] [General]
im so happy i found this using a sph 3600 and sph tl8000 both hybrid with battery and its working so well, forwarding to home-assist via mqtt i get data every time now, before using growatt integration...........

how do i buy you some coffes ? :)

**Comment by johanmeijer** [comment]
Thank you. I like it very much to hear people are using and are satisfied with Grott.

You can buy me some coffees when you are in Amsredam or sent a small donation via PayPal (link button available on Grott github start page). I would really appreciate it, but do not feel obliged.
